### DISCUSSION TITLE
"Tracked in" link no longer appearing in tracked issues

### DISCUSSION THREAD

**Discussion by john-craft** [community/community#48396] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

I noticed a problem recently where child issues that are tracked by a parent issue do not contain the "Tracked in" information at the top of the page. [Here is the relevant documentation](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists#navigating-tracked-issues) on navigating back to tracking issues. Parent tracking issues are fine and contain the task list count. It's only child issues that seem to be unaware (in the UI) that they are tracked by a parent. Is anyone else experiencing this?

**Comment by niklasastrom** [comment]
> Is anyone else experiencing this?

yes! I didn't realise how often I used this feature until it was gone 😲 

I hope it is just a temporary bug and that it is soon back.

**Comment by AlexRuiz7** [comment]
We used in for better navigation between parent (epic) and children issues. It made it very easy to see that a specific issue was part of something bigger.

I hope it comes back soon! I'm totally missing it now

**Comment by jbw976** [comment]
Yes we are seeing the same in the [Crossplane project](https://github.com/crossplane/crossplane).  In addition to not seeing the very helpful "[Tracked by](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-tasklists)" link in the header of issues, we also do not see "Tracks" and "Tracked by" fields as option to make visible within our projects, as described in [this documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/understanding-fields/about-tracks-and-tracked-by-fields).

These "tracked" fields seem to be missing from the [Crossplane org](https://github.com/crossplane), which is an open source community using a free tier of Github, but they are still visible within my company's github org, which is using a paid tier.

Is free vs. paid a potential cause of this difference?

**Comment by azenMatt** [comment]
Hi all - we are looking into what has caused this.

**Comment by azenMatt** [comment, marked as answer]
Hi everyone - apologies for the confusion this has caused. We pushed a fix out yesterday so please let me know if you are still having problems!

**Comment by jbw976** [comment]
Thanks for the update @azenMatt! I once again see the "Tracked by #xyz" information in the header of issues, e.g. https://github.com/crossplane/crossplane/issues/3807.  We appreciate that returning!

However, I still don't see the "tracked by" and "tracks" fields on project boards, as described in [this article](https://docs.github.com/en/issues/planning-and-tracking-with-projects/understanding-fields/about-tracks-and-tracked-by-fields).  An example project with this issue is https://github.com/orgs/crossplane/projects/20/views/1?filterQuery=tracked-by%3A&layout=table.  Following the article above, I don't seem to have those fields available.

Is this because we aren't in the tasklists beta? 🤔 

![Screenshot 2023-04-01 at 11 20 33 AM](https://user-images.githubusercontent.com/4313439/229308089-d8d008fa-080b-4e63-8864-86b828689ec0.png)

**Reply by john-craft** [reply]
> Is this because we aren't in the tasklists beta?

Yes, I think so.

**Reply by adminorcail** [reply]
Is it still in beta 5 months later or has the Tracked by field returned on your end? I still can't see it, and would very much appreciate the ability to group my subtasks with the parent issues

**Reply by PatrickKa** [reply]
Any update on this? I still do not see any hidden fields like "tracks" or "tracked by".

![image](https://github.com/community/community/assets/10672607/4dc503da-6c70-4cc2-ba35-8da9192ba895)

**Reply by jbw976** [reply]
@PatrickKa perhaps because you were not in the Tasklists beta? It looks like they won't be accepting more folks into the beta now as they are working on a different solution/direction: https://github.com/orgs/community/discussions/39106#discussioncomment-7532593 

Excited to see what comes from that - I've quite liked tasklists as they are in the beta, so if they can make them even better that would be cool! 😎

**Reply by PatrickKa** [reply]
I kind of hoped that the beta phase might be over a year later, but it does not seem like it. Fingers crossed that everyone can get access to those features soon 🤞.

**Comment by HebeHe1986** [comment]
yea, I really liked the "Tracked by" field, when can I get to use it. lol
